fast-vm

```diff
@@ -966,9 +985,38 @@ case "$1" in
 				pmsg $P_ERROR "$3 not executable\n" $vm_number
 			fi
 		else
-			ssh -o GlobalKnownHostsFile=/dev/null -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no "root@192.168.$SUBNET_NUMBER.$vm_number"
+			create_sshkey_if_needed
+			ssh -i $FASTVM_USER_CONF_DIR/id_fastvm -o GlobalKnownHostsFile=/dev/null -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no "root@192.168.$SUBNET_NUMBER.$vm_number"
 		fi
 		;;
+	scp)
+		vm_number="$2"
+		if [ -z "$vm_number" ]; then pmsg $P_ERROR "VM number missing\n"; usage ssh; fi
+		if [[ $vm_number = *[\ ]* ]]; then handle_multiple_requests $@;	fi
+
+		check_vm_number "$vm_number" "inactive"
+		update_access_time "$vm_number"
+
+		wait_for_ssh "$vm_number"
+		sourcepath=$(echo "$3" | sed "s/\(.*\)\bvm:\(.*\)/\1root@192.168.$SUBNET_NUMBER.$vm_number:\2/")
+		destpath=$(echo "$4" | sed "s/\(.*\)\bvm:\(.*\)/\1root@192.168.$SUBNET_NUMBER.$vm_number:\2/")
```

Comment:
At least basic check that `$3` and `$4` are not empty would be good here, otherwise we receive the errors like
~~~
./fast-vm scp 100
cp: cannot stat '': No such file or directory

./fast-vm scp 100 /tmp/test
cp: cannot create regular file '': No such file or directory
~~~

Reply:
Ah, I meant to add that check!  Added in this version, thanks:
$ ./fast-vm scp 51
[51][err] scp requires a sourcepath and destination path to copy
$ ./fast-vm scp 51 source
[51][err] scp requires a sourcepath and destination path to copy
$ ./fast-vm scp 51 source path
[51][inf] checking the 192.168.22.51 for active SSH connection (ctrl+c to interrupt)
[51][inf] 
[51]SSH ready
$